### DISCUSSION TITLE
Play silence to keep client (Roon) connected when no music playing

### DISCUSSION THREAD

**Discussion by privacyguy123** [savonet/liquidsoap#2303] [User Support]
It seems this would require some sort of fancy multiple device allowing ALSA setup and some tinkering in the liquidsoap .liq file ... I can't seem to work it out myself after a Google about. The fallback file won't play as a fallback Roon takes over full control of the audio device meaning runng some sort of aplay loop reports "device in use."

**Comment by Kerosel** [comment]
Forgive me if you've already thought of this, I don't quite understand the question or how exactly the Roon client fits in...

But if you'd like to play silence when there's no source, have you tried mksafe()? It basically plugs your source into a fallback() with blank() as the fallback option. If the source ever stops (becomes unavailable), mksafe() will output silence until the source comes back.
